modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/portlet/DDMFormAdminPortlet.java

```diff
@@ -122,6 +128,13 @@ public void render(
 		super.render(renderRequest, renderResponse);
 	}
 
+	@Activate
+	protected void activate(Map<String, Object> properties) {
+		_ffDateTimeDDMFormFieldTypeConfiguration =
+			ConfigurableUtil.createConfigurable(
+				FFDateTimeDDMFormFieldTypeConfiguration.class, properties);
+	}
+
```

Comment:
@rodrigopaulino why didn't you create a `FFDateTimeDDMFormFieldTypeConfigurationActivator`? Can we reuse this logic when creating a new feature flag in another story?

Reply:
Sure we can. The "Activator" component is only made necessary when there's already an `activate` method in the class and you can't get the properties from it. See [this](https://github.com/liferay-forms/liferay-portal/blob/15ee083a549625186a875cd53bef8731c6a44974/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormTemplateContextFactoryImpl.java#L156) example. The only way to inject a configuration in this class is by adding a `@Reference` to another component that loads the configuration.
Does it make sense?